androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -81,48 +105,52 @@ fun EnhancedVideoPlayer(
         }
     }
 
-    Box(
-        modifier = Modifier
-            .clickable(
-                indication = null,
-                interactionSource = remember { MutableInteractionSource() },
-                onClick = { isControlsVisible = !isControlsVisible }
-            )
-            .testTag("VideoPlayerParent")
+    AnimatedContent(
+        targetState = orientation
```

Comment:
I think you don't need this animated content anymore

Reply:
Done!